src/lightning/pytorch/accelerators/tpu.py

```diff
@@ -11,30 +11,19 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-from typing import Any, Dict, List, Optional, Union
+from typing import Any, Dict
 
-import torch
-
-from lightning.fabric.accelerators.tpu import _parse_tpu_devices, _XLA_AVAILABLE
 from lightning.fabric.accelerators.tpu import TPUAccelerator as FabricTPUAccelerator
 from lightning.fabric.utilities.types import _DEVICE
 from lightning.pytorch.accelerators.accelerator import Accelerator
 
 
-class TPUAccelerator(Accelerator):
+class TPUAccelerator(Accelerator, FabricTPUAccelerator):
```

Comment:
Reusing code through this

Reply:
Cant we share the code through the utility functions?

Reply:
Do you think that would be better? The implementations match exactly

Reply:
I'm fine with it but we need to be careful with the inheritance problems as we have experienced in the past.